### ISSUE TITLE
Unable to do certain actions on Android 9 due to network policy

### ISSUE THREAD

**Issue by jpalczewski** [emansih/FireflyMobile#34]
**Firefly III Server Version:**
4.7.12
**Firefly III Mobile Version:** 
1.2.6
**Android Version:**  
9, latest one
**Device Information:**  

- BRAND: Mobile
- MODEL: Mi A2
--------------------------------------
In Android 9 there's a opt-out blockade while limiting to clear-text servers, but it also hits edge cases(f.e. self-hosted firefly that is behind traefik which deals with lets-encrypt certs) for self-hosting apps. Similar open-source app for wallabag also had that issue and there's on-going easy PR that fixes it( https://github.com/wallabag/android-app/pull/762 ). However, while in WB app it denied loging in, in FM it allows to login, allows to add new transaction, but error as in-app notification bubble/cloud appears("CLEARTEXT communication to <domain> not permitted by network security policy") while loading transactions. 

## Expected Behavior
Application loads content from server in all available scenarios.

## Current Behavior
Android refuses to connect to server, in detail - to do certain actions. 

## Steps to Reproduce
1. Install and configure application on Android 9 and above
2. Enter into transactions submenu

**Comment by emansih** [comment]
This is working as expected.  Use https just like #25

**Comment by jpalczewski** [comment]
I'm using: https://imgur.com/a/ZBrPlXM
I fully agree with you that financial data shouldn't fly over unencrypted data, but in my situations thing looks weird:
- retrieving categories and other things works
- other fails at security policy. 

Do those application functions works on different network APIs?

**Comment by emansih** [comment]
Now that _is an unintended behavior_. I will need your server logs when such requests are being made. For apache, it will be `/var/log/apache2/access.log` or `other_vhosts_access.log` depending on your config.

**Comment by emansih** [comment]
ok, i can replicate this bug on a Android 9 emulator

**Comment by jpalczewski** [comment]
It's a docker image, from logs(I suppose that is just access.log) it's like:
```
firefly_iii_app_1  | 172.27.0.10 - - [27/Feb/2019:00:07:13 +0100] "GET /api/v1/transactions/?start=2019-02-11&end=2019-02-11&type=withdrawal&page=1 HTTP/1.1" 301 709 "-" "xyz.hisname.fireflyiii"
firefly_iii_app_1  | 172.27.0.10 - - [27/Feb/2019:00:07:26 +0100] "GET /api/v1/categories HTTP/1.1" 200 2669 "-" "xyz.hisname.fireflyiii"
```

I'm glad to see that you find it out and probably fixed! Nice project, btw. 
Of course I'm open to test beta apk's if you provide one.

**Comment by emansih** [comment]
> api/v1/transactions/?start=2019-02-11&end=2019-02-11&type=withdrawal&page=1

Heh... You can notice that the URL is wrong.

**Comment by emansih** [comment]
Thanks for your report. Fixed in v1.2.7.
